lib/evaluateArithmetic.js

```diff
@@ -13,9 +13,15 @@ function evaluateArithmeticDFS(node) {
   let innerNodeStatus;
   if (NodeType.isParenthesis(node)) {
     innerNodeStatus = evaluateArithmeticDFS(node.content);
+    if (innerNodeStatus.hasChanged()) {
```

Comment:
yeah that's a good question

which did you find easier to understand from a "less familiar with the code" viewpoint when you looked over the code last week?